### DISCUSSION TITLE
No useful frames found when running force-lut-modis

### DISCUSSION THREAD

**Discussion by mhelleis** [davidfrantz/force#188] [Q&A]
(moved this from issues to discussions)

Hello,

I'm trying to create a custom WVDB for Landsat processing following the tutorial, however for some reason no useful Terra/Aqua data is found.

The command I'm using:
force-lut-modis coords-file.coo wvdb geometa modis_hdfs -d 20220201,20220419

The content of the coords_file.coo is
```
5.3953 47.5589
```
The output is always:

```
2022/02/01. do. Download TERRA geometa. Try # 10
Download AQUA geometa. Try # 10
  0.0% of useful frames.
2022/02/02. do. Download TERRA geometa. Try # 10
Download AQUA geometa. Try # 10
  0.0% of useful frames.
2022/02/03. do. Download TERRA geometa. Try # 10
```

The resulting tables all contain the same value: 5.3953 47.5589 9999.000000 9999.000000 0

Things I've checked:

* Using coordinates from the example provided in the tutorial makes no difference.
* Changing the date range makes no difference
* The token for LAADS DAAC is found

Do you have any idea why this could be?

**Comment by thielfab** [comment]
Did you manage to get `force-lut-modis` running by now? 
I am having the same issue (https://github.com/davidfrantz/force/discussions/216)

**Reply by mhelleis** [reply]
No, we just reverted to using the static climatology table for now which provides accurate enough results for our use case

**Comment by davidfrantz** [comment]
Similar to https://github.com/davidfrantz/force/discussions/216, I cannot reproduce this....

The first few days in your time range and coordinate also give me 0% data. 2022/02/05 is the first date that gives me a result. But it always works in the first try for me.

Your 10 failed tries indicate that this is rather a connection/authorization issue... Not really sure what to do here to help...

David
